plugins-scripts/check_uptime.pl

```diff
@@ -262,6 +274,7 @@ ()
 	print "-c (--critical)  = Min. number of uptime to generate critical alert ( w < c )\n";
 	print "-f (--for)       = Show uptime in a pretty format (Running for x weeks, x days, ...)\n";
 	print "-s (--since)     = Show last boot in yyyy-mm-dd HH:MM:SS format (output from 'uptime -s')\n";
+	print "-s (--days)      = Show uptime in days\n";
```

Comment:
this should be `-d` i guess?

Reply:
Oh my, it was late at night ;]. Thanks. Fixed!